lighthouse-cli/cli-flags.js

```diff
@@ -175,6 +180,9 @@ function getFlags(manualArgv) {
       .default('output', ['html'])
       .default('port', 0)
       .default('hostname', 'localhost')
+      .default('form-factor', 'mobile')
+      // TODO: Remove undefined defaults when yargs-parser is >= v10. https://github.com/yargs/yargs-parser/issues/116
```

Comment:
any day now? :D

but why is this one forced to undefined/why are we explicitly defining `screenEmulation.mobile` in yargs as boolean?

Reply:
wellll,, what started as a little thing has now grown and turned back on itself.  okay i think i dont need it. 

I now see that booleans can be done as `--screenEmulation.mobile` and `--no-screenEmulation.mobile`. I had first tried `--screenEmulation.mobile=false` but that sets it using a string of `"false"`, so obv that was a problem.

Assuming that a user not using the presets, they'll have to use it correctly.